website/content/api-docs/secret/pki.mdx

```diff
@@ -3321,29 +3359,70 @@ the CRL.
    and delta CRLs only executes once a minute; this prevents high system load
    but limits the granularity of the temporal options below.
 
+~> Note: The `unified_crl`, `unified_crl_on_existing_paths`, and
+   `cross_cluster_revocation` parameters are all Vault Enterprise only
+   functionality. While they appear in the responses on Vault OSS, they cannot
+   be enabled.
+
 | Method | Path              |
 | :----- | :---------------- |
 | `POST` | `/pki/config/crl` |
 
 #### Parameters
 
 - `expiry` `(string: "72h")` - The amount of time the generated CRL should be valid.
+
 - `disable` `(bool: false)` - Disables or enables CRL building.
+
 - `ocsp_disable` `(bool: false)` - Disables or enables the OCSP responder in Vault.
+
 - `ocsp_expiry` `(string: "12h")` - The amount of time an OCSP response can be cached for,
   (controls the NextUpdate field), useful for OCSP stapling refresh durations. Setting to 0
   should effectively disable caching in third party systems.
+
 - `auto_rebuild` `(bool: false)` - Enables or disables periodic rebuilding of
   the CRL upon expiry.
+
 - `auto_rebuild_grace_period` `(string: "12h")` - Grace period before CRL expiry
   to attempt rebuild of CRL. Must be shorter than the CRL expiry period.
+
 - `enable_delta` `(bool: false)` - Enables or disables building of delta CRLs
   with up-to-date revocation information, augmenting the last complete CRL.
   This option requires `auto_rebuild` to also be enabled.
+
 - `delta_rebuild_interval` `(string: "15m")` - Interval to check for new
   revocations on, to regenerate the delta CRL. Must be shorter than CRL
   expiry.
 
+- `cross_cluster_revocation` `(bool: false)` - Enable cross-cluster revocation
+  request queues. When a serial not issued on this local cluster is presented
+  to Vault via the [`/revoke` API](#revoke-certificate), it is replicated
+  across clusters and the cluster which issued that certificate will revoke
+  it if it is online.
+
+~> Note: Revocations performed using Proof of Possession (POP) or with Bring
+   Your Own Certificate (BYOC) are not replicated cross-cluster. To enable
+   cross-cluster BYOC based revocation, turn on `unified_crl`.
+
+~> Note: `cross_cluster_revocation` is a Vault Enterprise only feature.
+
+- `unified_crl` `(bool: false)` - Enables unified CRL and OCSP building. This
```

Comment:
Should we have a call out to the considerations page on the two main feature flags?

Reply:
We called this out on the CRL URLs section, is it useful to call it out multiple times?

Reply:
I suppose not, ignore this for now.